framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -428,6 +447,45 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
                 result.add(new Postcondition(expr, postcondAnno, anno));
             }
         }
+
+        return result;
+    }
+
+    /**
+     * Returns the set of repeatable postconditions.
+     *
+     * @param metaAnno set of post condition meta annotations
+     * @param annotationmirrorType the class type used to cast the element value
```

Comment:
This comment says how the value is used, but not what it means.  Focus on how a client should use the method, not on how the method uses the formal parameter in its implementation.

Reply:
@mernst Can you give a hint on how can I write its documentation? I have updated the methods and documentation but I am having trouble with this. Please help!!